src/fuselayer/fuselayer.py

```diff
@@ -286,24 +304,17 @@ def write(self, path, data, offset, fh):
         # logger.info(f"WRITE CALLED WITH ID {id}")
         ret = os.pwrite(fh, data, offset)
         new_size = offset + ret
-        self.update_metadata(path, new_size)
+        self.metadata[path][1] = new_size
```

Comment:
what is the structure of self.metadata?

Reply:
I think a better way to manage the data is to use a dict or a class. It is difficult to remember the meaning of each filed of a tuple.